micro-advanced/data/Database.php

```diff
@@ -83,14 +83,6 @@ public function __construct(string $host = "127.0.0.1", int $port = 3306, string
             return;
         }
 
-        if ($mysql instanceof MySQL) $this->con = $mysql;
```

Comment:
This didn't doesn't make sense to do so because it will be overriden when line 103 runs which is right after this method.